### PR TITLE
add lbrlabs-eks package

### PR DESCRIPTION
**Pull request by jaxxstorm**

### CODE REVIEW

File: community-packages/package-list.json [hunk 1]
```diff
@@ -227,6 +227,10 @@
     {
       "repoSlug": "pulumiverse/pulumi-matchbox",
       "schemaFile": "provider/cmd/pulumi-resource-matchbox/schema.json"
+    },
+    {
+      "repoSlug": "lbrlabs/pulumi-lbrlabs/eks",
```
Comment (EvanBoyle) [inline community-packages/package-list.json:232]: I believe this should be `lbrlabs/pulumi-lbrlabs-eks`

Comment (cnunciato) [reply community-packages/package-list.json:232]: @jaxxstorm You'll also need to change the `layout` param in your `_index.md` doc to `package`. (Apparently this was changed recently so just calling it out so you aren't surprised when the docs fail to render.) We should also get a bit more content into the examples in those docs before merging. (Examples look TBD-ish.)

**Comment by cnunciato** [comment]
@jaxxstorm How should users think about this package differently from https://www.pulumi.com/registry/packages/eks/?

File: community-packages/package-list.json (hunk 1 above)
Comment (jaxxstorm) [reply community-packages/package-list.json:232]: ah yes, good catch!

**Comment by jaxxstorm** [comment]
> @jaxxstorm How should users think about this package differently from [pulumi.com/registry/packages/eks](https://www.pulumi.com/registry/packages/eks/)?

@cnunciato this essentially my replacement for EKS which greatly reduces the complexity of the inputs, has provides an opinionated set of addons and mechanisms like an ingress controller and cert-manager

**Review by pierskarsenbarg — APPROVED** [review]

File: community-packages/package-list.json [hunk 2]
```diff
@@ -227,6 +227,10 @@
     {
       "repoSlug": "pulumiverse/pulumi-matchbox",
       "schemaFile": "provider/cmd/pulumi-resource-matchbox/schema.json"
+    },
+    {
+      "repoSlug": "lbrlabs/pulumi-lbrlabs-eks",
+      "schemaFile": "provider/cmd/pulumi-resource-lbrlabs-eks/schema.json"
```
Comment (lukehoban) [inline community-packages/package-list.json:233]: I don't see a schema.json in this location in the repo.  I'm not an expert on how this works, but does this work?

**Review by lukehoban — APPROVED** [review]

File: community-packages/package-list.json (hunk 2 above)
Comment (cnunciato) [reply community-packages/package-list.json:233]: Yeah, I believe this needs to be this:

```suggestion
      "schemaFile": "schema.yaml"
```

**Review by sean1588 — APPROVED** [review]

**Pull request merged**